example/mongoose-multiple-connections.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const session = require('express-session')
+const MongoStore = require('connect-mongo').default
+
+// App Init
+const app = express()
+const port = 3000
+
+// Starting Server
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`)
+})
+
+// Mongoose Connection
+const appDBConnection = mongoose
+  .createConnection(process.env.APP_DB_URI, {
```

Comment:
Since it is an example, I expect it can be run with minimal setup. Maybe changing `process.env.APP_DB_URI` to a hardcoded value instead?